app/graphql/mutations/user_languages/create_user_language.rb

```diff
@@ -0,0 +1,14 @@
+module Mutations
+  module UserLanguages
+    class CreateUserLanguage < ::Mutations::BaseMutation
+      argument :params, Types::Input::UserLanguageInputType, required: true
+      type Types::UserLanguageType
+
+      def resolve(params:)
+        user_language_params = Hash params
+        user_language_params[:fluency_level] = user_language_params[:fluency_level].to_i
```

Comment:
I had to add this line (9) in so that I could actually pass fluency level in as a string, but then I had to convert it to an integer when actually creating. I'm not sure why I couldn't pass in an integer from the start, I think I might need an extra set of eyes

Reply:
@lkriffell maybe that's okay because they will get passed as a string from the request?